src/utils/colorUtils.ts

```diff
@@ -18,6 +18,10 @@ export module Utils {
       return l1 > l2 ? l1 / l2 : l2 / l1;
     }
 
+    /**
+     * Returns a brighter copy of this color. Each channel is multiplied by 0.7 ^ -factor.
+     * Channel values are capped at the maximum value of 255, and the minimum value of 30.
```

Comment:
I'm not sure it's worth describing implementation details, since that makes the docs fragile to functionality changes.